src/Relude/Lifted/Exit.hs

```diff
@@ -23,22 +23,19 @@ import System.Exit (ExitCode)
 
 import qualified System.Exit as XIO
 
+
 -- | Lifted version of 'System.Exit.exitWith'.
 exitWith :: MonadIO m => ExitCode -> m a
 exitWith a = liftIO (XIO.exitWith a)
-{-# INLINE exitWith #-}
```

Comment:
The reasoning is that if you call `exit*` functions, your program terminates and doesn't do anything else. So it doesn't make much sense to `INLINE` such functions, we can rely on GHC to do the right stuff. It's a minor micro-optimisation because abuse of `INLINE` can slow down compilation, but here I think we don't need it.